MetaMask/metamask-mobile
# test: initial off-ramp test to show build quote screen

## **Description**

This is the initial test and foundation for Off-Ramp testing. It should go through the onboarding screen, select region, select payment method, and show the screen to build a Sell quote. 


## **Related issues**

https://consensyssoftware.atlassian.net/browse/RAMPS-1869


## **Manual testing steps**

This test is consistently passing:

```
 PASS  e2e/specs/ramps/offramp.spec.js (36.081 s)
  smokeAssets OffRamp
    ✓ should select Region and Payment Method to see the Build Sell Quote screen (16087 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        36.139 s
```

## **Screenshots/Recordings**


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
